oauth2_http/javatests/com/google/auth/oauth2/UserCredentialsTest.java

```diff
@@ -722,6 +722,8 @@ public void IdTokenCredentials_WithUserEmailScope_success() throws IOException {
     IdTokenCredentials tokenCredential =
         IdTokenCredentials.newBuilder().setIdTokenProvider(credentials).build();
 
+    tokenCredential.getRequestMetadata();
```

Comment:
This looks like a no-op. What's going on here?

Reply:
OK, need a comment to that effect to keep future devs from removing this since this is a non-standard naming convention and confusing

Reply:
added assert for null and a comment for the line